.github/workflows/publish-docker.yaml

```diff
@@ -0,0 +1,69 @@
+name: Publish docker image on main and releases
+
+on:
+  push:
+    branches:
+      - main
+  release:
+    types: [released]
+  workflow_dispatch:
+  schedule:
+    # Monday at 9am UTC
+    - cron: '0 9 * * 1'
+
+permissions:
+  id-token: write
+  contents: read
+  packages: write
+
+jobs:
+  package:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PAT }}
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: |
+            ghcr.io/${{ github.repository }}
+            tricksterproxy/trickster

```

Comment:
I notice in Docker Hub, there's a trickstercache user, a tricksterproxy user, and a tricksterio user, that all claim to own the trickster image. Are these all you? Did I even choose the right one in my pull request? :smile:

Reply:
I just switched this to trickstercache because I saw that's the user called out in the readme here.

Reply:
But is it actually all 3 of them? And if so, is there a reason besides something like a typo-squatting defense?